### PR TITLE
CRV Risk Parameters Update 20230621

### PR DESCRIPTION
**Pull request by yonikesel**
Discussion: https://governance.aave.com/t/arfc-chaos-labs-risk-parameter-updates-crv-aave-v2-ethereum-2023-06-15/13709

### CODE REVIEW

File: src/AaveV2CRVRiskParams_20230621/AAVE-V2-CRV-RISK-PARAMS-20230621.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: Chaos Labs Risk Parameter Updates - CRV Aave V2 Ethereum - 2023.06.15
+author: Chaos Labs (@ori-chaoslabs, @yonikesel)
+shortDescription: CRV Risk Parameter Updates Aave V2 Ethereum
+discussions: https://governance.aave.com/t/arfc-chaos-labs-risk-parameter-updates-crv-aave-v2-ethereum-2023-06-15/13709
+created: 2023-06-21
```
Comment (sakulstra) [inline src/AaveV2CRVRiskParams_20230621/AAVE-V2-CRV-RISK-PARAMS-20230621.md:6]:
```suggestion
```

File: src/AaveV2CRVRiskParams_20230621/AAVE-V2-CRV-RISK-PARAMS-20230621.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+title: Chaos Labs Risk Parameter Updates - CRV Aave V2 Ethereum - 2023.06.15
+author: Chaos Labs (@ori-chaoslabs, @yonikesel)
+shortDescription: CRV Risk Parameter Updates Aave V2 Ethereum
```
Comment (sakulstra) [inline src/AaveV2CRVRiskParams_20230621/AAVE-V2-CRV-RISK-PARAMS-20230621.md:4]:
```suggestion
```
these fields are no longer needed

File: src/AaveV2CRVRiskParams_20230621/AaveV2CRVRiskParams_20230621Test.t.sol [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    createConfigurationSnapshot('preAaveV2CRVRiskParams_20230621Change', AaveV2Ethereum.POOL);
+
+    // 2. execute payload
+    _executePayload(address(proposalPayload));
+
+    // 3. create snapshot after payload execution
+    createConfigurationSnapshot('postAaveV2CRVRiskParams_20230621Change', AaveV2Ethereum.POOL);
+
+    // 4. Verify payload:
+    ReserveConfig[] memory allConfigsAfter = _getReservesConfigs(AaveV2Ethereum.POOL);
```
Comment (sakulstra) [inline src/AaveV2CRVRiskParams_20230621/AaveV2CRVRiskParams_20230621Test.t.sol:36]:
```suggestion
```

File: src/AaveV2CRVRiskParams_20230621/AaveV2CRVRiskParams_20230621Test.t.sol [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    ReserveConfig[] memory allConfigsBefore = _getReservesConfigs(AaveV2Ethereum.POOL);
+
+    // 1. create snapshot before payload execution
+    createConfigurationSnapshot('preAaveV2CRVRiskParams_20230621Change', AaveV2Ethereum.POOL);
+
+    // 2. execute payload
+    _executePayload(address(proposalPayload));
+
+    // 3. create snapshot after payload execution
+    createConfigurationSnapshot('postAaveV2CRVRiskParams_20230621Change', AaveV2Ethereum.POOL);
```
Comment (sakulstra) [inline src/AaveV2CRVRiskParams_20230621/AaveV2CRVRiskParams_20230621Test.t.sol:33]:
```suggestion
    ReserveConfig[] memory allConfigsAfterReserveConfig[] memory allConfigsAfter = createConfigurationSnapshot('postAaveV2CRVRiskParams_20230621Change', AaveV2Ethereum.POOL);
```

File: src/AaveV2CRVRiskParams_20230621/AaveV2CRVRiskParams_20230621Test.t.sol [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    _selectPayloadExecutor(AaveGovernanceV2.SHORT_EXECUTOR);
+  }
+
+  function testPayload() public {
+    AaveV2CRVRiskParams_20230621 proposalPayload = new AaveV2CRVRiskParams_20230621();
+
+    ReserveConfig[] memory allConfigsBefore = _getReservesConfigs(AaveV2Ethereum.POOL);
+
+    // 1. create snapshot before payload execution
+    createConfigurationSnapshot('preAaveV2CRVRiskParams_20230621Change', AaveV2Ethereum.POOL);
```
Comment (sakulstra) [inline src/AaveV2CRVRiskParams_20230621/AaveV2CRVRiskParams_20230621Test.t.sol:27]:
```suggestion
    ReserveConfig[] memory allConfigsBefore = createConfigurationSnapshot('preAaveV2CRVRiskParams_20230621Change', AaveV2Ethereum.POOL);
```

File: src/AaveV2CRVRiskParams_20230621/AaveV2CRVRiskParams_20230621Test.t.sol [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 17533013);
+    _selectPayloadExecutor(AaveGovernanceV2.SHORT_EXECUTOR);
+  }
+
+  function testPayload() public {
+    AaveV2CRVRiskParams_20230621 proposalPayload = new AaveV2CRVRiskParams_20230621();
+
+    ReserveConfig[] memory allConfigsBefore = _getReservesConfigs(AaveV2Ethereum.POOL);
```
Comment (sakulstra) [inline src/AaveV2CRVRiskParams_20230621/AaveV2CRVRiskParams_20230621Test.t.sol:24]:
```suggestion
```

File: src/AaveV2CRVRiskParams_20230621/AaveV2CRVRiskParams_20230621Test.t.sol [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+    // 2. execute payload
+    _executePayload(address(proposalPayload));
+
+    // 3. create snapshot after payload execution
+    createConfigurationSnapshot('postAaveV2CRVRiskParams_20230621Change', AaveV2Ethereum.POOL);
+
+    // 4. Verify payload:
+    ReserveConfig[] memory allConfigsAfter = _getReservesConfigs(AaveV2Ethereum.POOL);
+
+    ReserveConfig memory CRV_UNDERLYING_CONFIG = ProtocolV2TestBase._findReserveConfig(
```
Comment (sakulstra) [inline src/AaveV2CRVRiskParams_20230621/AaveV2CRVRiskParams_20230621Test.t.sol:38]:
```suggestion
    ReserveConfig memory CRV_UNDERLYING_CONFIG = _findReserveConfig(
```

File: src/AaveV2CRVRiskParams_20230621/AaveV2CRVRiskParams_20230621Test.t.sol [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+      allConfigsBefore,
+      AaveV2EthereumAssets.CRV_UNDERLYING
+    );
+
+    CRV_UNDERLYING_CONFIG.liquidationThreshold = CRV_LIQUIDATION_THRESHOLD;
+
+    CRV_UNDERLYING_CONFIG.ltv = CRV_LTV;
+
+    ProtocolV2TestBase._validateReserveConfig(CRV_UNDERLYING_CONFIG, allConfigsAfter);
```
Comment (sakulstra) [inline src/AaveV2CRVRiskParams_20230621/AaveV2CRVRiskParams_20230621Test.t.sol:47]:
```suggestion
    _validateReserveConfig(CRV_UNDERLYING_CONFIG, allConfigsAfter);
```

File: src/AaveV2CRVRiskParams_20230621/AaveV2CRVRiskParams_20230621.sol [hunk 9]
```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import {AaveV2Ethereum, AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+import {AaveV2PayloadEthereum, IEngine, EngineFlags} from 'aave-helpers/v2-config-engine/AaveV2PayloadEthereum.sol';
+import {ILendingPoolConfigurator} from 'aave-address-book/AaveV2.sol';
```
Comment (brotherlymite) [inline src/AaveV2CRVRiskParams_20230621/AaveV2CRVRiskParams_20230621.sol:6]:
```suggestion
```

**Pull request merged**